launcher/Application.h

```diff
@@ -310,4 +311,5 @@ private slots:
     QList<QUrl> m_zipsToImport;
     QString m_instanceIdToShowWindowOf;
     std::unique_ptr<QFile> logFile;
+    std::mutex loggerMutex;
```

Comment:
maybe you could put it in the `.cpp` file as a static variable? the Application header is included pretty much everywhere, so it would be cool if we could keep the things on it to a minimum.